### PR TITLE
Uses .isEmpty instead of .count > 0 for checking collections for emptiness

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

**Comment by rxwei** [comment]
Thanks!

File: MiniGo/Strategies/MCTS/MCTSNode.swift [hunk 1]
```diff
@@ -155,33 +155,29 @@ extension MCTSNode {
...
         }
 
-        let candidateCount = candidateIndexes.count
-        assert(candidateCount > 0)
-
         // Breaks the tie randomly.
-        let candidateIndex = candidateIndexes[Int.random(in: 0..<candidateCount)]
-        return elements[candidateIndex]
+        assert(!elements.isEmpty)
+        return elements.randomElement()!
```
Comment (xiejw) [inline MiniGo/Strategies/MCTS/MCTSNode.swift]: This line is wrong. Can you file it or rollback?

Comment (xiejw) [reply MiniGo/Strategies/MCTS/MCTSNode.swift]: s/file/fix

Comment (valeriyvan) [reply MiniGo/Strategies/MCTS/MCTSNode.swift]: @xiejw, why do you think it is wrong?

Comment (xiejw) [reply MiniGo/Strategies/MCTS/MCTSNode.swift]: The new change returns a random element from `elements`. It ignored the `candidateIndexes`. 

The old change picked up an `index` from `candidateIndexes` and returned the element from   `elements` according to that `index`.

So, the logic is different.

Comment (xiejw) [reply MiniGo/Strategies/MCTS/MCTSNode.swift]: I have fixed the tests at here 
https://github.com/tensorflow/swift-models/pull/121#pullrequestreview-229665641

**Pull request merged**